ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -85,9 +87,11 @@
   // the total number of data chunks retrieved so far (and may or may not have been written out yet).
   private int numChunksRetrieved;
   // the maximum size of a data chunk in bytes
-  private int chunkSize = CompositeBlobInfo.UNDEFINED_CHUNK_SIZE;
+  private long chunkSize;
```

Comment:
In the case of a simple blob, I set `chunkSize` to `totalSize`, to simplify calculation of the offset in `filterChunkToRange(..)`.  However, there is a rare possibility that a simple blob could be over 2 gb in size, in which case, it won't fit into an `int`.  Maybe we can just cast to an `int` in this case, since this has almost 0 probability of happening